src/Query/Builder.php

```diff
@@ -113,6 +114,8 @@ class Builder extends BaseBuilder
      */
     public $hint;
 
+    private ReadPreference $readPreference;
```

Comment:
I don't get why other properties are public. Using private by default to not add the new property to the API for no reason.